src/test/java/us/kbase/test/assemblyhomology/config/AssemblyHomologyConfigTest.java

```diff
@@ -256,6 +265,7 @@ public void pathWithUserWithIgnoreIPNullLoggerWithFilters() throws Throwable {
 		assertThat("incorrect mongo user", cfg.getMongoUser(), is(Optional.of("userfoo")));
 		assertThat("incorrect mongo pwd", cfg.getMongoPwd().get(),
 				equalTo("somepwd".toCharArray()));
+		assertThat("incorrect retry writes", cfg.getMongoRetryWrites(), is(true));
```

Comment:
Can you make some of the tests cover the cases where retry writes isn't missing, whitespace, or true?

Reply:
I updated some of the tests.